src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const countDays = days * 40;
+
+  if (days >= 7) {
+    return countDays - 50;
+  }
+
+  if (days >= 3) {
+    return countDays - 20;
+  }
+
+  if (days < 3) {
+    return countDays;
+  }
```

Comment:
The condition `if (days < 3)` is redundant. Since this is the last condition, you can simply return `countDays` without checking if `days < 3`. This will make the code cleaner and more efficient.